mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/tabs/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { __, _x } from '@wordpress/i18n';
+import { RoutedTabs } from '../../../../../../common/tabs/routed_tabs';
+import { Tab } from '../../../../../../common';
+import { AutomationFlow } from './automation_flow';
+import { Emails } from './emails';
+import { Orders } from './orders';
+import { Subscribers } from './subscribers';
+import { automationHasEmails } from '../../helpers/automation';
+
+export function Tabs(): JSX.Element {
+  const tabs = [
+    <Tab
+      key="automation-flow"
+      route="automation-flow/(.*)?"
+      title={__('Automation flow', 'mailpoet')}
+      automationId="tab-automation-flow"
+    >
+      <AutomationFlow />
+    </Tab>,
```

Comment:
This seems to create components inside the body of the parent. On each render, you'd get a completely new `Tab` component for each tab, which could get bad when there's a lot of content. If we need an array, for some reason, you can use `useMemo`, for instance.

Is the following not usable? (E.g., do those tabs forbid a `<>` as a child?)

```ts
<Tab ...>
{automationHasEmails() && (
   <>
     <Tab ...>
     <Tab ...>
     <Tab ...>
  </>
)}
```

Reply:
Ahh, an update — I see only now that this is using our old custom tabs. Could we use WP tabs instead? You can check the automation listing page, John implemented even some routing, if I'm not mistaken.